static/js/containers/pages/register/RegisterConfirmSentPage_test.js

```diff
@@ -47,13 +47,22 @@ describe("RegisterConfirmSentPage", () => {
 
   it("displays a link to create account page", async () => {
     const { inner } = await renderPage()
-    assert.equal(inner.find("li > a").prop("href"), routes.register.begin)
+    assert.equal(
+      inner
+        .find("Link")
+        .at(0)
+        .prop("to"),
+      routes.register.begin
+    )
   })
 
   it("displays user's email on the page", async () => {
     const { inner } = await renderPage()
     assert.equal(
-      inner.find(".confirm-sent-page > p > span").text("href"),
+      inner
+        .find(".auth-card > .col-12 > p > b")
```

Comment:
This highly specific selector could cause unnecessary problems if the HTML changes at all. I think it's better to just check that a string is included in the overall text in the card. Something like:

```es6
assert.include(
  inner.find(".auth-card").text(),
  `We sent an email to ${email}`
)